# MODULE_VERSION may *not* be above the latest Qt5 we package,
# refer to qt5-qtbase for current version.
--- a/.qmake.conf
+++ b/.qmake.conf
@@ -5,4 +5,4 @@ QTWEBENGINE_OUT_ROOT = $$shadowed($$PWD)
 load(qt_build_config)
 CONFIG += warning_clean
 
-MODULE_VERSION = 5.15.17
+MODULE_VERSION = 5.15.10
